DANS-KNAW/easy-validate-dans-bag
# EASY-2771: DC schema requires custom http agent

Fixes EASY-2771: DC schema requires custom http agent

#### When applied it will
* set the system property hhtp.agent with the configured value
* logs an error when the schema validators could not be created at startup
* 

#### Where should the reviewer @DANS-KNAW/easy start?

#### How should this be manually tested?

* disconnect from the web and restart the service, it fails but the log is not very informative
* reconnect to the web, build and deploy on deasy
* disconnect from the web and restart the service, it should log the failure to load a schema

#### Related pull requests on github
* [x] easy-dtap
